docs/spec/trustbloc-did-method.md

```diff
@@ -436,14 +454,15 @@ At this point, discovery and verification are complete. The client can now execu
   - Cache validated config files.
   - Present the list of endpoints parsed from said stakeholder configs, to be used for client DID method operations.
 
-### Error Cases
-Error cases which terminate the discovery process in a failure state:
-- Consortium config unavailable: The consortium domain points to a server that isn't functional
-- Insufficient endorsement: Insufficient stakeholders endorse the consortium
+## Security Considerations
```

Comment:
Is there any guidance we can add here to help clients using did:trustbloc when considering security implications? What do they need to do? What do they need to consider in their own implementations?

For example, what are (some of) the consequences of not checking the history files / signatures during discovery? How does a client check that the crypto algorithms meet their requirements before trusting the consortium? things like that.

The provided sections on crypto agility and TOFU are good.

Reply:
https://github.com/trustbloc/trustbloc-did-method/issues/71